install/kubernetes/templates/helm/istio/values.yaml.tmpl

```diff
@@ -89,3 +89,12 @@ security:
   imagePullPolicy: IfNotPresent
   resources: {}
   nodeSelector: {}
+
+  #
+  # addons configuration
+  #
+  addons:
```

Comment:
I like the structure - although in other parts of the PR I'm not sure how this is referenced.  (this is a pretty big PR, so a little difficult to review).